sys/include/net/nanocoap.h

```diff
@@ -12,6 +12,31 @@
  * @ingroup     net
  * @brief       Provides CoAP functionality optimized for minimal resource usage
  *
+ * # Create a Block-wise Response (Block2)
+ *
+ * Block-wise is a CoAP extension (RFC 7959) to divide a large payloads across
```

Comment:
payloads -> payload

Thanks for the update to use "block-wise". I had not really absorbed the use of the term in the RFC. I get the value of it now and agree we should use it similarly.

Reply:
> payloads -> payload

Fixed

> Thanks for the update to use "block-wise". I had not really absorbed the use of the term in the RFC. I get the value of it now and agree we should use it similarly.

I really didn't want to start stacking PR based change requests so I decided to push a fixup after merging your changes.

Reply:
Thanks for the update and no worries. Output is good and we're getting it done. :-)